internal/services/logic/logic_app_workflow_resource_test.go

```diff
@@ -170,6 +170,42 @@ func TestAccLogicAppWorkflow_accessControl(t *testing.T) {
 	})
 }
 
+func TestAccLogicAppWorkflow_identity(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_logic_app_workflow", "test")
+	r := LogicAppWorkflowResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.empty(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
```

Comment:
this is tested at the end, so by removing this we can check the Create path:

```suggestion
```

Reply:
Thanks.